### DISCUSSION TITLE
2005/11/02/%E6%88%91%E4%B8%80%E7%9B%B4%E6%8B%96%E7%9D%80%E6%B2%A1%E5%81%9A%E7%9A%84%E4%B8%A4%E4%BB%B6%E4%BA%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#52] [Announcements]
# 2005/11/02/%E6%88%91%E4%B8%80%E7%9B%B4%E6%8B%96%E7%9D%80%E6%B2%A1%E5%81%9A%E7%9A%84%E4%B8%A4%E4%BB%B6%E4%BA%8B

昨天在我的硬盘里乱翻，发现了以前从弟弟那里考来的一套有声教材《激发个人潜能》（安东尼罗宾）。于是，把第一课放在mp3里带回家听了听。那个作者真是能吹，说听完的这套课程，包我从此走向成功，说不定一不小心就成百万富翁了。

https://blog.qizhen.xyz/2005/11/02/%E6%88%91%E4%B8%80%E7%9B%B4%E6%8B%96%E7%9D%80%E6%B2%A1%E5%81%9A%E7%9A%84%E4%B8%A4%E4%BB%B6%E4%BA%8B

**Comment by ruanqizhen** [comment]
Lily
2007年06月3日 22:26 编辑
我上个月才拔,确实有用锤子敲,而且麻药醒的时候很疼.我以前有开过刀拉,都没有那么疼痛的记忆.真是恐怖!!

回复
Qizhen
2006年02月25日 21:09 编辑
到今天为止，我两件事都彻底解决啦  :D
 

回复
Jay
2005年11月3日 21:22 编辑
拔牙的时候，医生用锤子敲的哦:D

回复
竹贤
2005年11月3日 10:46 编辑
学习并非机械照搬照抄，而是从中发现自己尚未认识、尚未知晓、尚未试行过的他人的知识、经验和实践，结合自己的情况制定出自己的目标、方案和计划来，并且踏踏实实的执行。知道自己的不足，并非是进步的开始； 只有开始做了，并坚持了，才是开始。
